cmd/test/main.go

```diff
@@ -87,14 +87,19 @@ func New(ctx *TestCommandContext) *cobra.Command {
 
 func test(ctx *TestCommandContext, paths []string, flags TestCommandFlags) error {
 	messages := make(chan *messager.VersionMessage, 1)
+	go ctx.Messager.LoadVersionMessages(messages, ctx.CliVersion)
 	defer func() {
 		msg, ok := <-messages
 		if ok {
 			ctx.Printer.PrintMessage(msg.MessageText+"\n", msg.MessageColor)
 		}
 	}()
 
-	filePaths := ctx.Reader.FilterFiles(paths)
+	filePaths, err := ctx.Reader.FilterFiles(paths)
+	if err != nil {
+		fmt.Println(err.Error())
```

Comment:
we should add a more verbose error here, like what failed

Reply:
actually in this case the error is very straight forward: `no such file or directory`